src/Model/NoDriverGalleryManager.php

```diff
@@ -68,7 +67,7 @@ public function getTableName(): string
         throw new NoDriverException();
     }
 
-    public function getConnection(): Connection
+    public function getConnection()
```

Comment:
IMO, we should add a `NEXT_MAJOR` comment explaining the rationale about the missing return docblock or type declaration.

Reply:
I didn't add one because we will be able to change this when we want, not only on NEXT_MAJOR. This class is internal (and the other one too).

The rationale about this change is:

1. The retun typehint is not needed because the parent is not adding it.
2. We prefer not to add the typehint because that couples us with doctrine dbal and it is only used here
3. This method should also dissapear from the interface of the doctrine extension package because it is only needed for orm (and it can be solved in another way too, there is no need to have it)

Do you agree? want me to try a comment explaining that (not related to any NEXT_MAJOR tho)?

Reply:
Let me know what do you think of the message I added.